examples/subprotocol_server_youi/subprotocol_server_youi.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <vector>
+#define ASIO_STANDALONE
+#include <websocketpp/config/asio_no_tls.hpp>
+#include <websocketpp/server.hpp>
+
+typedef websocketpp::server<websocketpp::config::asio> server;
+
+using websocketpp::connection_hdl;
+using websocketpp::lib::placeholders::_1;
+using websocketpp::lib::placeholders::_2;
```

Comment:
I would ommit `_1` and `_2` so later in code its more explicit that its a placeholder arg.
```suggestion
using websocketpp::lib::placeholders;
```

so later on you can do `placeholders::_1`... I think it will be better. but that is just my opinion.

Reply:
yeah I agree, the way it is now is just copied from the other example